src/main/scala/NumberToLongShort.scala

```diff
@@ -0,0 +1,48 @@
+
+import scala.util.{Failure, Success, Try}
+
+class NumberToLongShort {
+
+  lazy val english: List[String] = List(" ", " thousand and ", " million ", " billion ", " trillion ", " quadrillion ", " quintillion ", " sextillion ")
+  lazy val french: List[String] = List(" ", " mille et ", " million ", " milliard ", " billion ", " billiard ", " trillion ", " triliard ")
+
+  def checkIfNumber(input: String, lang: List[String]): String = {
+    Try {
+      input.toLong
+    } match {
+      case Success(a) => langOutput(input, lang)
+      case Failure(a) => "Enter a valid positive number"
+    }
+  }
+
+  def langOutput(input: String, lang: List[String]): String = {
```

Comment:
Clean Code™ says that variable names and function names should't be abbreviated.